client/modules/IDE/hooks/useKeyDownHandlers.js

```diff
@@ -36,7 +36,7 @@ export default function useKeyDownHandlers(keyHandlers) {
     } else if (isCtrl) {
       handlers.current[`ctrl-${e.key.toLowerCase()}`]?.(e);
     }
-    handlers.current[e.key.toLowerCase()]?.(e);
+    handlers.current[e.key?.toLowerCase()]?.(e);
```

Comment:
Thanks for catching my mistake here! I assumed that keyboard event would also have an `e.key` but they do not. 🤦‍♀️ 

The `?.` on the previous two seem to have gotten overwritten when merging (Rachel's `else if` change).  Instead let's put `if (!e.key) return;` as the first line of the `handleEvent`.

Reply:
got it!